docs/release_notes/sat_2.6_release_notes.md

```diff
@@ -0,0 +1,73 @@
+# Changes in SAT 2.6
+
+The 2.6.4 version of the SAT product includes:
+
+- Version 3.24.0 of the `sat` python package and CLI.
+- Version 2.0.0-1 of the `sat-podman` wrapper script.
+- Version 1.6.1 of the `sat-install-utility` container image.
+
+## New `sat` commands
+
+No new `sat` commands were added in SAT 2.6.
+
+## Changes to `sat bootsys`
```

Comment:
Remember to add something about CRAYSAT-1722 like this from the CHANGELOG.md in sat:

> Fixed extreme slowness with the sat bootsys shutdown --stage platform-services command when a large SSH known_hosts file is in use.